apps/studio/src/features/editing-experience/components/form-builder/renderers/controls/JsonFormsProseControl.tsx

```diff
@@ -97,7 +97,7 @@ export function JsonFormsProseControl({
       editor.commands.setContent(data)
       editor.commands.setTextSelection(selection)
     }
-  }, [data])
+  }, [data, editor?.commands, editor?.state.selection])
```

Comment:
actually i think this MIGHT cause regression - im not very sure. Might need to do manual testing